web/server/web_server.c

```diff
@@ -137,6 +137,5 @@ void web_client_initialize_connection(struct web_client *w) {
     web_server_log_connection(w, "CONNECTED");
 
     web_client_cache_verify(0);
-}
-
 
+}
```

Comment:
I think we shouldn't change this file since we did not add any new code. It shouldn't be in the commit log cause it would be repository pollution.